packages/shopware-6-client/src/services/contextService.ts

```diff
@@ -0,0 +1,75 @@
+import { Currency } from "../interfaces/models/system/currency/Currency";
+import { apiService } from "../apiService";
+import { config } from "../settings";
+import {
+  getContextCurrencyEndpoint,
+  getContextCountryEndpoint,
+  getContextPaymentMethodEndpoint,
+  getContextShippingMethodEndpoint,
+  getContextLanguageEndpoint
+} from "../endpoints";
+import { Country } from "../interfaces/models/system/country/Country";
+import { ShippingMethod } from "../interfaces/models/checkout/shipping/ShippingMethod";
+import { PaymentMethod } from "../interfaces/models/checkout/payment/PaymentMethod";
+import { Language } from "../interfaces/models/framework/language/Language";
+import { SearchResult } from "../interfaces/response/SearchResult";
+
+const getCurrencies = async function(): Promise<SearchResult<Currency[]>> {
+  const resp = await apiService.get(
+    config.endpoint + getContextCurrencyEndpoint()
```

Comment:
please use template strings like this
```suggestion
    ${config.endpoint}${getContextCurrencyEndpoint()}
```

Reply:
Done :)